src/fal/cli/dbt_runner.py

```diff
@@ -66,13 +69,18 @@ def dbt_run(args, models_list: List[str]):
             command_list += ["--model"] + models_list
         else:
             command_list += ["--select"] + models_list
+
     if args.selector:
         command_list += ["--selector", args.selector]
 
-    command_list = list(map(str, command_list)) # make sure all are strings before joining for printing
+    command_list = list(
+        map(str, command_list)
+    )  # make sure all are strings before joining for printing
 
     # Execute the dbt CLI command in a subprocess.
-    full_command = " ".join(command_list) # make sure all are strings before joining for printing
+    full_command = " ".join(
+        command_list
+    )  # make sure all are strings before joining for printing
```

Comment:
you can actually remove the comment from THIS line and leave it on the above one.

Reply:
and remove the comment here
```suggestion
    full_command = " ".join(command_list)
```